.ci/scripts/gather_test_models.py

```diff
@@ -24,6 +24,7 @@
         "w2l": "linux.12xlarge",
         "ic4": "linux.12xlarge",
         "resnet50": "linux.12xlarge",
+        "llava_encoder": "linux.4xlarge",
```

Comment:
To run this on PR, please look for `model_should_run_on_event` and add `llava_encoder` to the list there.  This is the list of models we want to cover on PR, the rest are only run in trunk after committing (to save some CI resources a while ago)